packages/gamut/src/Form/FormGroupLabel.tsx

```diff
@@ -48,29 +47,22 @@ const labelSizeVariants = variant({
   },
 });
 
-const disabledLabelStyle = ({ disabled }: disabledLabelStyleProps) => {
-  if (disabled) {
-    return css`
-      color: ${theme.colors['gray-600']};
-    `;
-  }
-};
+const labelColorStates = states({
+  disabled: {
+    ...formFieldTextDisabledStyles,
+  },
+});
```

Comment:
Fixed the mutability issue here so this should be safe again :) 

```suggestion
const labelColorStates = states({
  disabled: formFieldTextDisabledStyles,
});
```

Reply:
🙏